### ISSUE TITLE
`<button>` with `title` is not considered to have an accessible name

### ISSUE THREAD

**Issue by clottman** [microsoft/accessibility-insights-web#917]
## Describe the bug

Accessibility Insights reports a button doesn't have an accessible name when the accessible name is set with `title`. 

**To Reproduce**

1. Create a page that has `<button class='some css with background image' title="an accessible name"></button>` on it.
2. Run the automated tests from the extension
3. See that the extension reports a failure for WCAG 4.1.2, button does not have accessible text.

## Expected behavior
`<button>` with `title` set should not be flagged as not having accessible text.

## Extension (please complete the following information)
- OS: Windows
- Version: Accessibility Insights for Web 2.6.0 (axe-core 3.2.2)
- Browser Version: Chrome version 75.0.3770.100
- Target Page: internal

## Are you willing to submit a PR?
Maybe? I've never contributed before. 

## Did you search for similar existing issues?
yes

## Additional context
[Technique H91: Using HTML form controls](https://www.w3.org/WAI/WCAG21/Techniques/html/H91)
According to the [W3 spec for the Accessibility API](https://www.w3.org/TR/2019/WD-html-aam-1.0-20190703/#button-element-accessible-name-computation), `title` is in the list of things to consider for the accessible name of a `<button>` 

## Failure details from the extension: 

Title: WCAG 4.1.2: Buttons must have discernible text (iframe;.pswp__button--arrow--left)
Tags: Accessibility, WCAG 4.1.2, button-name

Issue: Buttons must have discernible text (button-name: https://dequeuniversity.com/rules/axe/3.2/button-name?application=msftAI)

Snippet: `<button class="pswp__button pswp__button--arrow--left" title="Previous"> </button>`

**Comment by iamrafan** [comment]
hi @clottman , thanks for reporting the issue. This is a false positive caused because of the "space" inside the ```<button>``` element. 

We are going to update axe-core to version 3.3.0 pretty soon. It contains PR https://github.com/dequelabs/axe-core/pull/1615 which fixes this issue

**Comment by karanbirsingh** [comment]
hi @clottman - we've updated to `axe-core 3.3.2`. In the latest build I don't see this issue anymore on this [codepen](https://codepen.io/karansin/pen/wvvabzL). If you still see this after updating, do let us know. Thanks for reporting! cc @ferBonnin

**Comment by ferBonnin** [comment]
Marking this as resolved, please let us know if you encounter this issue again and we can reactivate this bug :)
